contracts/libs/utils/TokenBalance.sol

```diff
@@ -57,7 +59,7 @@ library TokenBalance {
     }
 
     function sendFunds(IERC20 token, address receiver, uint256 amount) internal {
-        token.safeTransfer(receiver, amount.from18(address(token)));
+        token.safeTransfer(receiver, amount.from18Safe(address(token)));
```

Comment:
This will revert if amount is < than token decimals. Potentially forever locking the contract